libs/zedUpload/awsutil/s3sync.go

```diff
@@ -8,24 +8,28 @@ import (
 	"compress/gzip"
 	"fmt"
 	"io"
+	"math"
 	"net/http"
 	"os"
 	"path/filepath"
+	"sync"
 	"sync/atomic"
 	"time"
 
 	"github.com/aws/aws-sdk-go/aws"
 	"github.com/aws/aws-sdk-go/aws/awserr"
 	"github.com/aws/aws-sdk-go/service/s3"
 	"github.com/aws/aws-sdk-go/service/s3/s3manager"
+	"github.com/lf-edge/eve/pkg/pillar/types"
```

Comment:
Please define the struct (is it downloadedparts?) in zedupload and import that into pillar.